mapbox/libandroid-ui/src/main/java/com/mapbox/services/android/ui/geocoder/GeocoderAdapter.java

```diff
@@ -408,11 +396,11 @@ public Filter getFilter() {
 
   private class GeocoderFilter extends Filter {
 
-    private final MapboxGeocoding.Builder builder;
+    private final MapboxGeocoding.Builder<MapboxGeocoding.Builder> builder;
 
-    public GeocoderFilter() {
+    GeocoderFilter() {
       super();
-      builder = new MapboxGeocoding.Builder();
+      builder = new MapboxGeocoding.Builder<>();
```

Comment:
Was receiving a lint warning `unchecked assignment` so I added the `<>` which seemed to make it happy. Can switch back if there's an issue?